README.md

```diff
@@ -105,6 +105,23 @@ In case you use podman for testcontainers, some tips must be taken into account:
 
 ---
 
+## Field injection and Parameter Resolution
```

Comment:
Thank you for updating the docs.  Not for action now - wonder if this kind of thing should be package doc?

Reply:
I can add it in the morning and merge after assuming your happy with a copy and paste of this (give or take formatting)

Reply:
Second thoughts - maybe extending the javadoc on KafkaClusterExtension would be a better home for this text?

Reply:
Added it to `KafkaClusterExtension` (in 450c030824cca96ceda8acba2cb221dc535de0a2) as that is the natural entry point for users. 

It fits well with the existing notes.